Source/Core/VideoBackends/Vulkan/ShaderCache.cpp

```diff
@@ -1279,8 +1280,12 @@ void ShaderCache::PrecompileUberShaders()
 
 void ShaderCache::WaitForBackgroundCompilesToComplete()
 {
-  m_async_shader_compiler->WaitUntilCompletion();
+  m_async_shader_compiler->WaitUntilCompletion([](size_t completed, size_t total) {
+    Host_UpdateProgressDialog("Compiling shaders...", static_cast<int>(completed),
```

Comment:
Please use `GetStringT` for this string and the equivalent ubershader strings, so they can be translated.